rollingpin/args.py

```diff
@@ -21,12 +32,17 @@ def __call__(self, parser, namespace, values, option_string):
 
 def _add_selection_arguments(config, parser):
     selection_group = parser.add_argument_group("host selection")
+
+    default_hosts = config["deploy"].get("default-hosts", [])
+    if not isinstance(default_hosts, list):
+        default_hosts = [default_hosts]
+
     selection_group.add_argument(
         "-h",
         action=ExtendList,
         nargs="+",
-        default=[],
-        required=True,
+        default=default_hosts,
+        required=len(default_hosts) == 0,
```

Comment:
Thanks for the explanation! Could you add this as an in-line comment? 

I'm not sure on general style but IMO wrapping the expression in `()` makes it a bit more readable. I don't care too much though.